Makefile

```diff
@@ -215,6 +214,7 @@ help:
 	@echo " - generate-swagger                 generate the swagger spec from code comments"
 	@echo " - swagger-validate                 check if the swagger spec is valid"
 	@echo " - golangci-lint                    run golangci-lint linter"
+	@echo " - golangci-lint                    run golangci-lint linter with --fix"
```

Comment:
```suggestion
	@echo " - golangci-lint-fix                run golangci-lint linter with --fix"
```

Reply:
Thanks, I think I will rethink these lint targets because eslint and stylelint also have `--fix` so a general approach is needed to cover all three.